src/index.js

```diff
@@ -1,86 +1,89 @@
-const db = require('./notifications/db');
-const Configuration = require('./notifications/config');
-const notificationController = require('./notifications/controller');
+import db from './notifications/db';
+import Configuration from './notifications/config';
+import notificationController from './notifications/controller';
 
-function Notification(config) {
-  this.config = new Configuration(config);
-  this.configDatabase = this.config.database;
-  this.baseApi = this.config.baseApi;
-}
+class Notification {
+  constructor(config) {
+    this.config = new Configuration(config);
+    this.configDatabase = this.config.database;
+    this.baseApi = this.config.baseApi;
+  }
 
-Notification.prototype.startServer = (app) => {
-  const self = this;
-  self.app = app;
-  self.io = app.io;
-  db.connect(this.configDatabase);
+  startServer(app) {
+    const self = this;
+    self.app = app;
+    self.io = app.io;
+    db.connect(this.configDatabase);
 
-  // Expose endpoints
-  self.app.get(`${self.baseApi}/all/:userid`, notificationController.getAll);
-  self.app.put(`${self.baseApi}/:notificationId`, notificationController.updateUnread);
-  self.app.delete(`${self.baseApi}/:notificationId`, notificationController.delete);
-  self.app.post(`${self.baseApi}/all`, notificationController.updateAllUnread);
+    // Expose endpoints
+    self.app.get(`${self.baseApi}/all/:userid`, notificationController.getAll);
+    self.app.put(`${self.baseApi}/:notificationId`, notificationController.updateUnread);
+    self.app.delete(`${self.baseApi}/:notificationId`, notificationController.delete);
+    self.app.post(`${self.baseApi}/all`, notificationController.updateAllUnread);
 
-  self.io.on('connection', (socket) => {
-    socket.emit('who-are-you');
+    self.io.on('connection', (socket) => {
+      socket.emit('who-are-you');
 
-    socket.on('check-in', (user, ack) => {
-      self.io.sockets[user.id] = socket;
-      self.sendUnread(user.id);
+      socket.on('check-in', (user, ack) => {
+        self.io.sockets[user.id] = socket;
+        self.sendUnread(user.id);
 
-      // Send to client endpoint base route forexposed endpoints
-      ack(`{"baseApi":"${self.baseApi}"}`);
+        // Send to client endpoint base route forexposed endpoints
+        ack(`{"baseApi":"${self.baseApi}"}`);
```

Comment:
Maybe a space after `:`? Just to keep the consistency of our style guide (Dumb comment)